visivo/parsers/line_validation_error.py

```diff
@@ -0,0 +1,61 @@
+from pydantic import ValidationError
+from typing import List
+from pathlib import Path
+import yaml
+import re
+
+from visivo.parsers.yaml_ordered_dict import YamlOrderedDict
+
+
+def find_line_string_start(file_path, input_dict):
+    with open(file_path, "r") as file:
+        file_content = file.read()
+
+    file_lines = re.sub(r"^[ \t-]+", "", file_content, flags=re.MULTILINE).split("\n")
+    input_line = re.sub(
+        r"^[\s-]+", "", yaml.dump(input_dict), flags=re.MULTILINE
+    ).split("\n")[0]
+    for line_number, file_line in enumerate(file_lines):
+        if input_line in file_line:
+            return line_number + 1
+    return -1
+
+
+class LineValidationError(Exception):
```

Comment:
@tgsoverly made updates to the message that results in these changes. I think it helps readability, but curious what your thoughts are: 

Old Message: 
![Screenshot 2024-01-04 at 8 40 17 AM](https://github.com/visivo-io/visivo/assets/35309169/ac865fb2-887f-4f52-a1e5-05788ae3922d)
New Message:
![Screenshot 2024-01-04 at 8 39 08 AM](https://github.com/visivo-io/visivo/assets/35309169/b376928b-c2ef-410c-b5f3-86d6c8370700)